requirements.txt

```
@@ -1,6 +1,6 @@
 psutil==5.9.7
 PySide6==6.6.1
 PySide6-Addons==6.6.1
-suntime==1.2.5
+suntime>=1.2.5
```

This needs 1.3.0 at least

---

Ah, you’re right, they added the timezone parameter.

```suggestion
suntime>=1.3.1
```